src/instructlab/data/generate.py

```diff
@@ -155,6 +156,12 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--gpus",
+    type=click.IntRange(min=0),
```

Comment:
Would you mind changing this for eval as well so they are consistent.

Reply:
done